### PR TITLE
get method bugfix

### PR DESCRIPTION
**Pull request by o100ja**
This was probably a copy-paste from the list method that simply was overlooked.

### CODE REVIEW

**Review by esatterwhite — CHANGES_REQUESTED** [review]
Ya, it kind of looks that way. Would you mind adding a simple test for this?

can really just be a create w/ a get & verify the data matches. That would be awesome!

File: lib/index.js [hunk 1]
```diff
@@ -236,7 +236,7 @@ King = new Class({
      **/
     ,get:co.wrap(function* list( resource, id ){
         let resp = yield this.request('get', resource, id);
-        return (resp.body) && resp.body.data;
+        return resp.body;
```
Comment (esatterwhite) [inline lib/index.js:239]: we can do away with the variable assignment here too and just
```js
return (yeild this.request('get', resource, id)).body;
```

**Comment by o100ja** [comment]
I've extended the test a bit to cover the change I've added, but I simply can't find the time to do a full CRUD test, especially for each API endpoint. Since I'll be relying on this library on a project I'm working on currently (and relentlessly :sweat:) I'll try to extend the tests a bit more when I accumulate enough time to work on that.

**Review by esatterwhite — APPROVED** [review]

**Comment by esatterwhite** [comment]
Thanks! Look forward to anymore additions that may come.

**Comment by esatterwhite** [comment]
2.0.1 published to npm

**Pull request merged**